app/models/post.rb

```diff
@@ -10,6 +10,6 @@ def recent_comments
   end
 
   def update_posts_counter
-    user.increment!(:Posts_Counter)
+    author.increment!(:Posts_Counter)
```

Comment:
- [ ] Kindly change this to `PostsCounter`, based on what you have in the table as highlighted by the previous reviewer.